common/lib/xmodule/xmodule/partitions/tests/test_partitions.py

```diff
@@ -205,12 +223,14 @@ def test_version_upgrade(self):
         }
         user_partition = UserPartition.from_json(jsonified)
         self.assertEqual(user_partition.scheme.name, "random")    # pylint: disable=no-member
+        self.assertEqual(user_partition.parameters, {})
```

Comment:
We should probably add a test case for the version upgrade from 2 --> 3 as well.